apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner/Migrations/Operations/SpannerCreateDatabaseOperation.cs

```diff
@@ -15,13 +15,11 @@
 namespace Microsoft.EntityFrameworkCore.Migrations.Operations
 {
     /// <summary>
-    /// 
     /// </summary>
     public class SpannerCreateDatabaseOperation : MigrationOperation
     {
         /// <summary>
```

Comment:
Nit: Empty summary

Reply:
There is a big dearth of comments throughout.  I'll do a followup PR with more comments -- but you won't find many if any comments in any of the providers.

Reply:
Also just a quick note -- virtually no classes are used directly from user code -- so the comments only affect our own maintenance.  The only exception to this is custom metadata, which currently Spanner has none.

Reply:
SGTM